researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/BridgeDataProvider.java

```diff
@@ -50,7 +51,9 @@
 * This is a very simple implementation that hits only part of the Sage Bridge REST API
 * a complete port of the Sage Bridge Java SDK for android: https://github.com/Sage-Bionetworks/BridgeJavaSDK
  */
-public abstract class BridgeDataProvider extends DataProvider {
+@AnyThread
+@Deprecated
```

Comment:
Good idea.   The way were going I feel like we are going to have to release RS as RS2 😃